ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/behaviorTree/scene/RDXBehaviorTreeSceneObject.java

```diff
@@ -35,10 +35,7 @@ public RDXBehaviorTreeSceneObject(long id, CRDTInfo crdtInfo, IsaacROSFoundation
       gizmo.createAndSetupDefault(baseUI.getPrimary3DPanel());
       gizmo.getPoseGizmo().setGizmoFrame(referenceFrame);
 
-      String modelName = "";
-      if (objectType.equals(IsaacROSFoundationPoseObject.MUSTARD))
-         modelName = "environmentObjects/mustard/mustard.glb";
-
+      String modelName = "environmentObjects/" + objectType.meshName + "/" + objectType.meshName + ".glb";
```

Comment:
These meshes don't exist, you'll crash the app I think. I only made it for the mustard so far.